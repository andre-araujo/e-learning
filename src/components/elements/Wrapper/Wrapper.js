import React from 'react';
import { node, string } from 'prop-types';

import {
  Container,
} from './styles';

function Wrapper({
  children,
  className,
  margin,
}) {
  return (
    <Container className={className} margin={margin}>
      {children}
    </Container>
  );
}

Wrapper.defaultProps = {
  children: null,
  className: null,
};

Wrapper.propTypes = {
  children: node,
  className: string,
};

export default Wrapper;
