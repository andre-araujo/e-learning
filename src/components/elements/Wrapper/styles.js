import styled from 'styled-components';

export const Container = styled.div`
    max-width: 980px;
    padding: 0 30px;
    margin: 0 auto;
`;
