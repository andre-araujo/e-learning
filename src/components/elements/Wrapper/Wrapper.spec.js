import React from 'react';
import { shallow } from 'enzyme';

import Wrapper from './Wrapper';

describe('Wrapper component', () => {
  it('should match snapshot', () => {
    expect(shallow(<Wrapper />)).toMatchSnapshot();
  });
});
