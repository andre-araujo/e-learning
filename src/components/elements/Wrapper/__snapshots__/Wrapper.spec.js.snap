// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wrapper component should match snapshot 1`] = `
<styled.div
  className={null}
/>
`;
